README.md

```diff
@@ -0,0 +1,35 @@
+# Awesome Cryptocurrencies
+
+## 5th Cycle: 2024 - 2028
+
+### Top Dogs
+
+- DOT
+- RUNE
+- ATOM
+- GNO
+- AR (Formerly used by SOL as data layer, now they switched to google cloud. Is AR still a top contender?)
```

Comment:
(Formerly used by SOL as data layer, now they switched to google cloud. Is AR still a top contender?)